### PR TITLE
Update "Getting Started" to match latest Android Studio

### PR DESCRIPTION
**Pull request by gaearon**
The "Configure" menu includes a dropdown:

screen shot 2017-02-13 at 20 43 57

I also clarified the UI role of labels (a tab and a checkbox) because I didn't notice them at first.

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -198,19 +198,19 @@ Click "Next" to install all of these components, then [configure VM acceleration
 
 #### 3. Install the Android 6.0 (Marshmallow) SDK
 
-Android Studio installs the most recent Android SDK by default. React Native, however, requires the `Android 6.0 (Marshmallow)` SDK. To install it, launch the SDK Manager, click on "Configure" in the "Welcome to Android Studio" screen.
+Android Studio installs the most recent Android SDK by default. React Native, however, requires the `Android 6.0 (Marshmallow)` SDK. To install it, launch the SDK Manager, click on "Configure" > "SDK Manager" in the "Welcome to Android Studio" screen.
 
 > The SDK Manager can also be found within the Android Studio "Preferences" menu, under **Appearance & Behavior** → **System Settings** → **Android SDK**.
 
-Select "SDK Platforms" from within the SDK Manager, then check the box next to "Show Package Details". Look for and expand the `Android 6.0 (Marshmallow)` entry, then make sure the following items are all checked:
+Select "SDK Platforms" tab from within the SDK Manager, then check the box next to "Show Package Details" in the bottom right corner. Look for and expand the `Android 6.0 (Marshmallow)` entry, then make sure the following items are all checked:
```
Comment (hramos) [inline docs/GettingStarted.md]: Small nit:

> Select *the* "SDK Platforms" tab

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -198,19 +198,19 @@ Click "Next" to install all of these components, then [configure VM acceleration
...
 
 - `Google APIs`
 - `Android SDK Platform 23`
 - `Intel x86 Atom System Image`
 - `Intel x86 Atom_64 System Image`
 - `Google APIs Intel x86 Atom_64 System Image`
 
-Next, select "SDK Tools" and check the box next to "Show Package Details" here as well. Look for and expand the "Android SDK Build Tools" entry, then make sure that `Android SDK Build-Tools 23.0.1` is selected.
+Next, select "SDK Tools" tab and check the box next to "Show Package Details" here as well. Look for and expand the "Android SDK Build Tools" entry, then make sure that `Android SDK Build-Tools 23.0.1` is selected.
```
Comment (hramos) [inline docs/GettingStarted.md]: Same here. Add "the" before "SDK Tools".

**Comment by hramos** [comment]
Some very minor nits, thanks for the PR! We can merge once those are addressed.

**Comment by gaearon** [comment]
Done

**Pull request closed without merging**